packages/api-db-mongodb/src/migration.ts

```diff
@@ -895,6 +895,45 @@ export const Migrations: Migration[] = [
       const invoices = db.collection(CollectionName.Invoices)
       await invoices.updateMany({}, {$set: {manuallySetAsPaidByUserId: undefined}})
     }
+  },
+  {
+    version: 22,
+    async migrate(db) {
```

Comment:
Prisma does not support number enums. Migrated to string enums

Reply:
Bad Prisma, bad ;)